app/helpers/copy_helper.rb

```diff
@@ -0,0 +1,6 @@
+module CopyHelper
+  BP_CONTROLLED_NUMERATOR_COPY = "Patients with BP <140/90 at their most recent visit in the last 3 months"
```

Comment:
This is a good start to consolidating all this, but you may want to move this to Rails builtin i18n library -- check out https://guides.rubyonrails.org/i18n.html#abstracting-localized-code for an example.

That would also make it easy to translate to other languages in the future if necessary.

Reply:
Niceee that's so cool! Thanks @rsanheim! Will delete the `CopyHelper` module and add the numerator and denominator copy to the `en.yml` file